src/components/UI/molecules/UploadBox/ImageUploadBox.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from "@emotion/styled";
+import { useS3Upload } from "next-s3-upload";
+import { useState } from "react";
+import { Button } from "src/components/UI/atoms";
+import { deletefiles } from "src/api/file";
+
+const ImageUploadBox = () => {
+  const [imageUrl, setImageUrl] = useState<string>("");
+  const [imageKey, setImageKey] = useState<string>("");
+  const { FileInput, openFileDialog, uploadToS3 } = useS3Upload();
+
+  const handleFileChange = async (file: File) => {
+    const { url, key }: { url: string; key: string } = await uploadToS3(file);
```

Comment:
요렇게 해도될것 같은데 아닌가요? uploadToS3 함수의 return 타입이 any 인가요?? 그게 아니라면 타입 자동추론이 될 듯해서요

```suggestion
    const { url, key } = await uploadToS3(file);
```
그리고 요건 try catch 문으로 감싸는게 좋을 것 같다는 생각도 드네요

Reply:
감사합니다!